### ISSUE TITLE
problems with large blocks of images

### ISSUE THREAD

**Issue by francoflamigni** [cdcseacave/openMVS#95]
OpenMVS with openMVG for the alignement works really very well!
The only problem is that for large blocks it is not able to terminate.
My computer has 16 GB Ram, 4 core 3Ghz Intel i5 processor.
For example I have a block of 200 16Mpixels (4920 x 3264) images. I use reduction level of 1 ( depth map calculated for 2460 x 1632) during the fusion step it exits with failure (catching an exception).
I try the same block with level 2 (1230 x 816) and in 40 minutes it produces an excellent 35 Mpoints cloud.
Is there a way to work even if the ram is not enought?

Thank you
Franco Flamigni (Italy)

**Comment by cdcseacave** [comment]
yes, there are ways to make OpenMVS support huge datasets, but are not implemented yet... this is on the enhancements list, and any help in coding is welcome...

**Comment by cdcseacave** [comment]
Closing as stale. The OpenMVS codebase has undergone extensive refactoring and modernisation on the \develop\ branch since this issue was opened. If you believe this is still relevant, please test against the latest \develop\ branch and open a new issue with reproduction steps.
